src/Auth.ts

```diff
@@ -485,6 +496,12 @@ export class Auth {
     });
   }
 
+  private calculateThumbprint(certificate: pki.Certificate) {
+    const messageDigest = md.sha1.create();
```

Comment:
Let's decorate the const with type to avoid refactoring issues in the future.

Reply:
Done